config/crds/faros_v1alpha1_clustergittrackobject.yaml

```diff
@@ -6,6 +6,13 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: clustergittrackobjects.faros.pusher.com
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.conditions[0].status
+    name: In-Sync
```

Comment:
Thoughts on making this `InSync`? To me it would feel more consistent with the naming of the columns for the GitTrack resource.